src/upscaler.py

```diff
@@ -703,7 +706,16 @@ def run(self):
 
                 # downscale frames with Lanczos
                 Avalon.info(_('Lanczos downscaling frames'))
-                shutil.rmtree(self.extracted_frames)
+                try:
+                    shutil.rmtree(self.extracted_frames)
+                except Exception:
```

Comment:
Exception is too broad. You might be catching that's not intended to be caught here.
Use `except PermissionError` to be specific.